### ISSUE TITLE
Windows 10 hubot: Cannot load adapter flowdock

### ISSUE THREAD

**Issue by kjata30** [flowdock/hubot-flowdock#95]
Getting this error when running a clean install of hubot (from npm) using the flowdock adapter.

C:\Hubot> .\/bin/hubot -a flowdock -d

[Tue Jan 24 2017 16:06:54 GMT-0500 (Eastern Standard Time)] DEBUG Loading adapter flowdock
[Tue Jan 24 2017 16:06:55 GMT-0500 (Eastern Standard Time)] ERROR Cannot load adapter flowdock - TypeError: Cannot read
property 'prototype' of undefined

project.json

  "dependencies": {
    "hubot": "latest",
    "hubot-diagnostics": "latest",
    "hubot-flowdock": "latest",
    "hubot-google-images": "latest",
    "hubot-google-translate": "latest",
    "hubot-help": "latest",
    "hubot-heroku-keepalive": "latest",
    "hubot-maps": "latest",
    "hubot-pugme": "latest",
    "hubot-redis-brain": "latest",
    "hubot-rules": "latest",
    "hubot-scripts": "latest",
    "hubot-shipit": "latest"
  },
  "engines": {
    "node": "7.40",
	"npm": "4.0.5"
  }

python 2.7.13
node-gyp 3.5.0

**Comment by mrala** [comment]
Have you tried a newer version of node or `npm rebuild`?

**Comment by kjata30** [comment]
No, but I ended up just reverting to nodejs 0.12.x and it works just fine.

**Comment by bhebing** [comment]
I do have this issue, but I'm a bit confused about versioning and node. I think I have the latest, but node -v reports as v7.4.0. I did do a rebuild.

Should I upgrade something, or actually downgrade?

**Comment by bhebing** [comment]
OK, so the versioning is clear now :) 

Anyway, issue is on v7.4.0, but on the v6 LTS release.

**Comment by kjata30** [comment]
@bhebing Downgrading to node 0.12.18 solved this issue for me.
